mappings/net/minecraft/block/entity/ChestBlockEntity.mapping

```diff
@@ -7,12 +7,20 @@ CLASS bto net/minecraft/block/entity/ChestBlockEntity
 	METHOD a getPlayersLookingInChestCount (Lbgq;Lev;)I
 		ARG 0 world
 		ARG 1 pos
-	METHOD a recalculateViewerCountIfNecessary (Lbhf;Lbtf;III)I
+	METHOD a countViewers (Lbhf;Lbtf;III)I
 		ARG 0 world
 		ARG 1 container
 		ARG 2 ticksOpen
 		ARG 3 x
 		ARG 4 y
+	METHOD a updateViewerCountIfNecessary (Lbhf;Lbtf;IIIII)I
```

Comment:
updateViewerCount? Seems implied, that no updates happen when it's not necessary.

Reply:
By "if necessary", I mean if `ticksOpen + x + y + z` is a multiple of 200. It's called every tick, but only updates every 200 ticks for efficiency. Naming it `updateViewerCount` would be confusing since it doesn't always update it. Maybe `tickViewerCount` would be better.